openff/interchange/smirnoff/_virtual_sites.py

```diff
@@ -178,6 +181,12 @@ def store_potentials(  # type: ignore[override]
                     ),
                 },
             )
+
+            logger.info(
+                "Charge section VirtualSites applied to virtual site with orientation atoms "
```

Comment:
```suggestion
                "Charge section VirtualSites applied to virtual site with orientation atoms at topology indices "
```

I'm not sure this is a huge improvement but I feel like clarifying what the atom numbers are is helpful.